projects/developer/src/app/processing/jobs/component.html

```diff
@@ -93,7 +93,7 @@ <h2><i class="fa fa-cube"></i> Jobs <span *ngIf="count">({{ count }})</span></h2
                 </div>
                 <div *ngSwitchCase="'inputs'">
                     <div *ngIf="rowData.input_files">
-                        <span tooltipStyleClass="max-size" [pTooltip]="makeTooltip(rowData.input_files)">

```

Comment:
Can you also remove this from the .ts file? It doesn't look like it's used anywhere else.

Reply:
Fixed thanks for pointing it out